base/loading.jl

```diff
@@ -229,17 +228,14 @@ function _require_search_from_serialized(node::Int, mod::Symbol, sourcepath::Str
             return restored
         end
     end
-    if isa(restored, Exception)
-        warn("Deserialization checks failed while attempting to load cache from $failedpath.")
-        warn(restored, prefix="WARNING: ")
-    end
     return !isempty(paths)
 end
 
 # to synchronize multiple tasks trying to import/using something
 const package_locks = Dict{Symbol,Condition}()
 
 # used to optionally track dependencies when requiring a module:
+const DEBUG_LOADING = Ref(false)
```

Comment:
comment directly above doesn't really apply to this constant, only the following ones